src/components/GoalsComponents/GoalSublist/GoalSublist.scss

```diff
@@ -11,7 +11,7 @@
 
 .sublist-title {
   margin-left: -10px;
-  text-transform: capitalize;
+  // text-transform: capitalize;
```

Comment:
Why not remove it completely? Leaving it commented out will make anyone including tour future self wonder why - and if it should be uncommented ;).

Reply:
Sorry my bad 
Sure will remove this